src/juv/_run_template.py

```diff
@@ -75,7 +75,7 @@ def as_with_arg(self) -> str:
 juv_data_dir = Path(user_data_dir("juv"))
 juv_data_dir.mkdir(parents=True, exist_ok=True)
 
-temp_dir = tempfile.TemporaryDirectory(dir=juv_data_dir)
+temp_dir = tempfile.TemporaryDirectory(dir=juv_data_dir, ignore_cleanup_errors=True)
```

Comment:
It looks like `ignore_cleanup_errors` [was added](https://github.com/python/cpython/pull/24793) in Python 3.10, so some of the tests are failing.

I'd like to support whatever is not EOL, so maybe we could just make a little subclass with the behavior we want:

```python
# Custom TemporaryDirectory for Python < 3.10
# TODO: Use `ignore_cleanup_errors=True` in Python 3.10+
class TemporaryDirectoryIgnoreErrors(tempfile.TemporaryDirectory):
    def cleanup(self):
        try:
            super().cleanup()
        except Exception:
            pass  # Ignore cleanup errors
```

and just copy that code snippet into the two files where it is used.